src/http/api/reply.ts

```diff
@@ -89,13 +90,12 @@ export async function handleCreateReply(
     }
 
     const to = PUBLIC_COLLECTION;
-    const cc = [attributionActor, apCtx.getFollowersUri(ACTOR_DEFAULT_HANDLE)];
 
     const conversation = objectToReplyTo.replyTargetId || objectToReplyTo.id!;
     const mentions = [
         new Mention({
             href: attributionActor.id,
-            name: attributionActor.name,
+            name: getHandle(attributionActor),
```

Comment:
This wasn't really useful as it was not adding the handle here but just the name. See tag [here](https://browser.pub/https://princivershwal.ghost.io/.ghost/activitypub/note/c1c0ce3b-6dac-48f9-8ecc-ff502b21efe6)